vue-training/src/services/api.js

```diff
@@ -0,0 +1,7 @@
+import { create } from 'apisauce'
+
+const api = create({
+  baseURL: 'http://localhost:3002'
```

Comment:
use a .env

Reply:
Added `API_URL='http://localhost:3002` in `.env` file